SPECS-EXTENDED/cassandra/cassandra.spec

```
@@ -0,0 +1,209 @@
+BuildRequires:  xerces-j2
+BuildRequires:  python3
+BuildRequires:  python3-devel
+BuildRequires:  python3-libs
+BuildRequires:  python3-setuptools
+
+Requires:       msopenjdk-11
+Requires:       javapackages-tools
+Requires:       which
+Requires:       gawk
+Requires:       shadow-utils
+Requires(post): /bin/chown
+%{?systemd_requires}
+
+%description
+Cassandra is a highly scalable, eventually consistent, distributed, structured key-value store.
+Cassandra brings together the distributed systems technologies from Dynamo and the log-structured storage engine from Google's BigTable.
+
+%prep
+%autosetup -p1 -n apache-%{name}-%{version}-src
+
+%build
+# refer to Readme file for 
+# cassandra-build-cache
+cp %{SOURCE2} .
+tar -xvf  %{SOURCE2}
+mkdir -p ~/.m2
+mv repository ~/.m2/
+
+export JAVA_HOME=$(echo /usr/lib/jvm/msopenjdk-11)
+export ANT_OPTS="-Xmx1024m -XX:MaxPermSize=512m"
+ant -v clean jar javadoc -Drelease=true -Duse.jdk11=true
+
+%install
+mkdir -p %{buildroot}%{_var}/opt/%{name}/data
+mkdir -p %{buildroot}%{_var}/log/%{name}
+mkdir -p %{buildroot}%{_bindir}
+mkdir -p %{buildroot}%{_sbindir}
+mkdir -p %{buildroot}%{_datadir}/cassandra
+mkdir -p %{buildroot}%{_sysconfdir}/cassandra
+mkdir -p %{buildroot}%{_sysconfdir}/sysconfig
+mkdir -p %{buildroot}%{_sysconfdir}/profile.d
+
+cp -pr conf/* %{buildroot}%{_sysconfdir}/cassandra/
+
+rm -f bin/cqlsh
+rm -f bin/cqlsh.py
+mv bin/%{name} %{buildroot}%{_sbindir}
+mv bin/%{name}.in.sh %{buildroot}%{_datadir}/cassandra/
+cp -p bin/* %{buildroot}%{_bindir}/
+cp -p tools/bin/* %{buildroot}%{_bindir}/
+cp -r lib %{buildroot}%{_var}/opt/cassandra/
+cp -r build %{buildroot}%{_var}/opt/cassandra/
+cp -p build/tools/lib/stress.jar %{buildroot}%{_var}/opt/cassandra/lib
+cp -p build/apache-cassandra-%{version}.jar %{buildroot}%{_var}/opt/cassandra/lib
+
+mkdir -p %{buildroot}%{_unitdir}
+install -p -D -m 644 %{SOURCE1}  %{buildroot}%{_unitdir}/%{name}.service
+
+cat >> %{buildroot}%{_sysconfdir}/sysconfig/cassandra <<- "EOF"
+CASSANDRA_HOME=%{_var}/opt/cassandra/
+CASSANDRA_CONF=%{_sysconfdir}/cassandra/
+EOF
+
+cat >> %{buildroot}%{_sysconfdir}/profile.d/cassandra.sh <<- "EOF"
+export CASSANDRA_HOME=%{_var}/opt/cassandra/
+export CASSANDRA_CONF=%{_sysconfdir}/cassandra/
+EOF
+
+%pre
+getent group cassandra >/dev/null || /usr/sbin/groupadd -r cassandra
+getent passwd cassandra >/dev/null || /usr/sbin/useradd --comment "Cassandra" --shell /bin/bash -M -r --groups cassandra --home /var/opt/%{name}/data cassandra
+
+%post
+%{_sbindir}/ldconfig
+chown -R cassandra: /var/opt/cassandra
+source /etc/profile.d/cassandra.sh
+%systemd_post cassandra.service
+
+%preun
+%systemd_preun cassandra.service
+
+%postun
+%{_sbindir}/ldconfig
+%systemd_postun_with_restart cassandra.service
+if [ $1 -eq 0 ] ; then
+    /usr/sbin/userdel cassandra
+    /usr/sbin/groupdel cassandra
+fi
+
+%files
+%defattr(-,root,root)
+%doc README.asc CHANGES.txt NEWS.txt conf/cqlshrc.sample LICENSE.txt NOTICE.txt
```

Please declare license-related files with the `%license` macro:

```suggestion
%license LICENSE.txt
%doc README.asc CHANGES.txt NEWS.txt conf/cqlshrc.sample NOTICE.txt
```

---

added %license LICENSE.txt